spec/serializers/va_facilities/geo_serializer_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'va_facilities/geo_serializer'
+
+RSpec.describe VaFacilities::GeoSerializer do
+  let(:vha_facility) { build :vha_648A4 }
+
+  context '::geometry' do
+    it 'makes a geometry object' do
+      geo_object = VaFacilities::GeoSerializer.geometry(vha_facility)
+
+      expect(geo_object.keys).to eq(%w[type coordinates])
+      expect(geo_object.values).to eq(['Point', [vha_facility.long, vha_facility.lat]])
+    end
+  end
+
+  context '::properties' do
+    it 'collects basic facility attributes' do
```

Comment:
Does this farm out to the facility serializer already tested above for each of these properties?

Reply:
No, the geo_serializer has a `properties` method which calls these attributes on the object being serialized.

Reply:
It would be nice to have the same testing of the values of these properties for the GeoSerializer as the FacilitySerializer has.

Reply:
Even though they use the same ApiSerializer module right now, that might not always be the case. It would be nice to explicitly make those assertions for Geo, too, so that we could change the implementation, if needed.